src/getText.py

```diff
@@ -1,32 +1,56 @@
 ## Function: getText
 ## Input: URL
-## Output: Unformatted Text extratcted from the URL
+## Output: Unformatted TEXT extracted from the URL and ARTICLE
 ## Author: The Sentimentalists / Ana B Potje
 ## Date: 24/Sep/2020
+## Last Modified: 22/Oct/2020
 ##
 ## This function uses the Python Library Newspaper, which reads the article from the URL,
-## remove some unwanted characters and words, and returns the unformatted text.
+## remove some unwanted characters and words, so the unformatted text is stored in a variable "text".
+## It then applies the "nlp()" function to perform natural language processing on the news article.
+##
+## The objects "text" and "article" are the output of this function.
+##
+## PS: Use of the "article" object:
+##   news_article = getArticle("https://www.bbc.co.uk/news/uk-54234084")
+##   news_article.top_image
+##   >>> https://ichef.bbci.co.uk/images/ic/1024x576/p08s2yp9.jpg
+##   news_article.keywords
+##   >>> ['action', '50000', 'patrick', 'face', 'covid19', 'restrictions', 'governments', ...]
+##   news_article.summary
+##   >>> Sir Patrick Vallance said that would be expected to lead to about "200-plus deaths per day" a 
+##       month after that. On Monday, a further 4,368 daily cases were reported in the UK, up from 3,899....
 ###########################################################################################################
 
-
 def getText(url):
-    from newspaper import Article
-    ### Getting the Text
+    from newspaper import Article 
+    import nltk
+    nltk.download('punkt')
+    ### Getting the ARTICLE
     try:
         article = Article(url)
         article.download()
         article.parse()
     ### Exception - e.g if URL is "valid" but inexistent, no text will be retrieved
     except Exception as e: 
         #print(e)
-        return('-1')
-    
+        return  {'text': '-1',
+                 'header': '', 
```

Comment:
Wonder if we should standardise these to make it easier on the frontend, and instead return an error object like:
`{"error" : "The article could not be downloaded." }`

Reply:
Maybe ignore this and it could be a future fix.